docker-compose/is-analytics/identity-server/repository/conf/carbon.xml

```diff
@@ -36,7 +36,7 @@
     <!--
        Product Version
     -->
-    <Version>5.3.0</Version>
+    <Version>5.4.1</Version>
```

Comment:
@deki Ideally we should be using the configuration file from 5.4.1 default pack for this and update that with any deployment specific configurations.